### PR TITLE
Added collation to columns

### PR DESCRIPTION
**Pull request by marcio-santos-zocdoc**
@Zocdoc/ci-devs 
@son-huynh-zocdoc 
- adds collation to varchar columns
- works with TABLE and TYPE

### CODE REVIEW

File: model/Models/Column.cs [hunk 1]
```diff
@@ -55,11 +55,22 @@ public Column(string name, string type, byte precision, int scale, bool nullable
...
-		}
-
-		public ColumnDiff Compare(Column c) {
+        public string RowGuidColText {
+	        get { return IsRowGuidCol ? "ROWGUIDCOL" : string.Empty; }
+        }
+
+        public string Collation { get; set; }
+
+        public string CollationText {
```
Comment (son-huynh-zocdoc) [inline model/Models/Column.cs:64]: Nitpick but `Collation` and `CollationText` could just be one property with a backing field.

**Comment by son-huynh-zocdoc** [comment]
Seems fine to me :+1:

**Comment by colin-hanson-zocdoc** [comment]
lgtm

**Comment by son-huynh-zocdoc** [comment]
@marcio-santos-zocdoc Any idea when we can deploy this change? I'd like to merge my PR but it currently still shows as failing in CleanDB.

**Pull request merged**